src/test/java/seedu/address/testutil/AnimalBuilder.java

```diff
@@ -112,6 +122,17 @@ public AnimalBuilder withAdmissionDate(String admissionDate) {
         return this;
     }
 
+    /**
+     * Sets the {@code taskList} of the {@code Animal} that we are building.
+     *
+     */
+    public AnimalBuilder withTaskList(List<Task> taskList) {
+        this.taskList = new TaskList();
+        this.taskList.getTaskList().addAll(taskList);
```

Comment:
uh, implement an `addAll` that does not require the internal List of taskList to be exposed.

Reply:
Noted :)